saleor/dashboard/group/views.py

```diff
@@ -37,7 +40,8 @@ def group_create(request):
     return TemplateResponse(request, 'dashboard/group/detail.html', ctx)
 
 
-@superuser_required
+@staff_member_required
+@permission_required('userprofile.edit_group')
```

Comment:
I added `edit` permissions to this view, as I see no use case, when user may want to access it without intention to make some changes, `group_list` seems to be sufficient for viewing. @koradon @elwoodxblues is that ok?

Reply:
Yes, thats correct. We have that permissions schema on other 'lists' and 'details' views. The only exceptions are 'detail' views that are not rendering forms but just static pages.